client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementMeta.vue

```diff
@@ -185,6 +185,32 @@
           :options="submitTypeOptions"
           @select="(val) => emitInput('submitType', val)" />
 
+        <template v-if="hasPermission('field_statement_phase')">
+          <dp-select
+            v-if="hasPermission('field_show_internal_procedure_phases_in_dropdown') && !localStatement.attributes.isSubmittedByCitizen"
+            id="statementProcedureInternalPhase"
+            v-model="localStatement.attributes.phase"
+            class="mb-3"
+            :disabled="!editable || !localStatement.attributes.isManual"
+            :label="{
+              text: Translator.trans('procedure.public.phase')
+            }"
+            :options="availableInternalPhases"
+            @select="(val) => emitInput('phase', val)" />
+
+          <dp-select
+            v-else
+            id="statementProcedureExternalPhase"
+            v-model="localStatement.attributes.phase"
+            class="mb-3"
+            :disabled="!editable || !localStatement.attributes.isManual"
+            :label="{
+              text: Translator.trans('procedure.public.phase')
+            }"
+            :options="availableExternalPhases"
+            @select="(val) => emitInput('phase', val)" />
+        </template>
```

Comment:
If these stay like that (basically the same code), I think we should merge them into one and dynamically change the parts that are different (`options`)